job_executor/config/log.py

```diff
@@ -30,7 +23,7 @@ def format(self, record: logging.LogRecord) -> str:
                     record.created,
                     tz=datetime.timezone.utc,
                 ).strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3]
-                + "Z",
+                              + "Z",
```

Comment:
ruff did this? 🤔  feels like an intellij thing

Reply:
Yes, it's Ruff plugin that misbehaves in Intellij I think, I'll fix 🏃